lib/routes/account.js

```diff
@@ -1233,39 +1231,36 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
                   request.emitMetricsEvent('account.deleted', {uid})
                 ])
               })
-              .then(() => {
-                return {}
-              })
           }, (err) => {
             if (err.errno === error.ERRNO.ACCOUNT_UNKNOWN) {
-              customs.flag(request.app.clientAddress, {
+              return customs.flag(request.app.clientAddress, {
                 email: form.email,
                 errno: err.errno
               })
             }
             throw err
           })
-          .then(reply, reply)
+
       }
     }
   ]
 
   if (config.isProduction) {
-    delete routes[0].config.validate.payload.preVerified
+    delete routes[0].options.validate.payload.preVerified
   } else {
     // programmatic account lockout was only available in
     // non-production mode.
     routes.push({
       method: 'POST',
       path: '/account/lock',
-      config: {
+      options: {
         validate: {
           payload: true
         }
       },
-      handler: function (request, reply) {
+      handler: async function (request, h) {
```

Comment:
...same here.

Reply:
@deeptibaghel did you miss this one ?

Reply:
@vladikoff this one is done, found a number of other cases, have done those in the next PR.